forge-gui/res/cardsfolder/upcoming/the_tale_of_tamiyo.txt

```diff
@@ -0,0 +1,14 @@
+Name:The Tale of Tamiyo
+ManaCost:2 U
+Types:Legendary Enchantment Saga
+K:Chapter:4:DBRepeat,DBRepeat,DBRepeat,DBChangeZone
+SVar:DBRepeat:DB$ Repeat | RepeatSubAbility$ DBCleanAndGrind | MaxRepeat$ MaxRepeats | RepeatCheckSVar$ MilledSharesType | RepeatSVarCompare$ GE2 | SubAbility$ DBCleanup | SpellDescription$ Mill two cards. If two cards that share a card type were milled this way, draw a card and repeat this process.
+SVar:DBCleanAndGrind:DB$ Cleanup | ClearRemembered$ True | SubAbility$ DBGrind
+SVar:DBGrind:DB$ Mill | NumCards$ 2 | NoReveal$ True | RememberMilled$ True | SubAbility$ DBDraw
+SVar:DBDraw:DB$ Draw | ConditionCheckSVar$ MilledSharesType | ConditionSVarCompare$ GE2
+SVar:MaxRepeats:Count$InYourLibrary
+SVar:MilledSharesType:Remembered$Valid Card.sharesCardTypeWithOther Remembered
+SVar:DBChangeZone:DB$ ChangeZone | Origin$ Graveyard | Destination$ Exile | TargetMin$ 0 | TargetMax$ Yard | TgtPrompt$ Select any number of target instant, sorcery, and/or Tamiyo planeswalker cards in your graveyard | ValidTgts$ Instant.YouOwn,Sorcery.YouOwn,Planeswalker.Tamiyo+YouOwn | RememberChanged$ True | SubAbility$ DBPlay | StackDescription$ SpellDescription | SpellDescription$ Exile any number of target instant, sorcery, and/or Tamiyo planeswalker cards from your graveyard. Copy them. You may cast any number of the copies.

```

Comment:
TargetMax SVar doesn't exist

Reply:
Now you're not really filtering by the targetable types
What actually happens in this case?

Reply:
Well, my test never got past the first chapter, so no idea -- sorry, I'm pretty tired here :(

Reply:
you should be able to reach IV now?

Reply:
any results?

Reply:
Still hangs on the first chapter.